src/components/Person/Person.jsx

```diff
@@ -1 +1,36 @@
-// export const Person = ({ person }) => ();
+const FEMALE = 'f';
+const HUSBAND = 'husband';
+const WIFE = 'wife';
+const NOT_MARRIED_STATUS_MESSAGE = 'I am not married';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {Boolean(age) && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      <p className="Person__partner">
+        {isMarried
+          ? `${partnerName} is my ${sex === FEMALE
+            ? HUSBAND
+            : WIFE}`
+          : NOT_MARRIED_STATUS_MESSAGE}
```

Comment:
Avoid using nested conditions

Reply:
Create variable for `${sex === FEMALE ? HUSBAND : WIFE}`

Reply:
Thanks for your comment
Fixed👌